### ISSUE TITLE
Can't create direct paging for a team

### ISSUE THREAD

**Issue by Hronom** [grafana/terraform-provider-grafana#1313]
### Terraform Version
* Terraform: v1.7.1
* Terraform Grafana Provider: 2.6.1
* Grafana: Grafana v10.4.0-65610 (85b9edcd28)

Unable to create direct paging for a team, getting error:
```
╷
│ Error: POST https://oncall-prod-us-central-0.grafana.net/oncall/api/v1/integrations/: 400 {detail: Direct paging integrations can't be created}
│ 
│   with module.team["xxx"].grafana_oncall_integration.team_direct_paging[0],
│   on modules/team/oncall-integration.tf line 1, in resource "grafana_oncall_integration" "team_direct_paging":
│    1: resource "grafana_oncall_integration" "team_direct_paging" {
│ 
╵
```

### Affected Resource(s)
Please list the resources as a list, for example:
* grafana_oncall_integration

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

```
resource "grafana_oncall_integration" "team_direct_paging" {
  provider = grafana.oncall_target

  name = "${var.name} oncall direct paging"
  type = "direct_paging"

  team_id = xxx.id
  default_route {
    slack {
      channel_id = xxx
      enabled    = true
    }
    escalation_chain_id = xxx.id
  }
}
```

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    grafana = {
      source  = "grafana/grafana"
      version = "2.6.1"
    }
    http = {
      source  = "hashicorp/http"
      version = "3.4.0"
    }
    null = {
      source  = "hashicorp/null"
      version = "3.2.1"
    }
  }

  backend "s3" {}
}
```

### Expected Behavior
What should have happened?

Possibility to create and manage direct_paging for newly created teams

### Actual Behavior
What actually happened?

Seems like during team creation there automatically created team direct paging.
However we need to manage by our-self direct paging and assign proper escalation policies.
Currently this not possible.

**Comment by FecoDoo** [comment]
Is there any release plan for the fix? The issue has been marked as completed but without further info.

**Comment by joeyorlando** [comment]
@FecoDoo yes! I'll be publishing the public documentation regarding this shortly, but in the mean-time there is a new org-setting in OnCall:

![Image](https://github.com/user-attachments/assets/70b0317b-8c88-4f05-ae54-1785b2bb2cb3)

Once you enable this it'll allow you to start managing your Direct Paging Integrations via Terraform. Existing Integrations won't be removed, and you can either delete these (only via the public API atm; we _may_ add web UI support to CRUD these, when this org setting is enabled, in the future), or `terraform import` them.

**Comment by joeyorlando** [comment]
this doesn't require you to bump your terraform provider version (this was strictly an OnCall backend change to support this). **note** that this functionality is only available on Grafana Cloud, if you're running Grafana OnCall OSS this isn't supported.

feedback on the feature would be welcome!

**Comment by FecoDoo** [comment]
> Once you enable this it'll allow you to start managing your Direct Paging Integrations via Terraform. Existing Integrations won't be removed, and you can either delete these (only via the public API atm; we _may_ add web UI support to CRUD these, when this org setting is enabled, in the future), or `terraform import` them.

That's great! However, is there any way to enable that toggle when initializing the oncall plugin with environmental variables or any other means which could be automated in some way?

My point is we might want to have zero manual operations and trust Terraform to fully configure the oncall plugin, which is not possible currently as one always needs to login into Grafana and click-here-and-there :)

Edited: sorry I didn't read the comment followed, would absolutely love to see this feature in OSS version as well.
